RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -64,7 +64,8 @@ public override void Load()
             Bind<Sinks>().ToSelf().InSingletonScope();
             Bind<App>().ToSelf().InSingletonScope();
             Bind<RubberduckParserState>().ToSelf().InSingletonScope();
-            Bind<GitProvider>().ToSelf().InSingletonScope();
+            Bind<ISourceControlProvider>().To<GitProvider>();
```

Comment:
There's a problem with that though: when a scope isn't specified, Ninject uses `TransientScope`, which doesn't dispose of `IDisposable` dependencies.

    public class GitProvider : SourceControlProviderBase, IDisposable

And `GitProvider` implements `IDisposable`, so `TransientScope` is going to cause issues one way or another.